packages/sakuli-legacy/src/init-command/create-structure.ts

```diff
@@ -0,0 +1,70 @@
+import { mkdirSync, writeFileSync } from "fs";
+import { stripIndent } from 'common-tags';
+
+export const createTestsuite = (directory: string, suiteName: string) => {
+    const rootDir = directory + `/${suiteName}`;
+    mkdirSync(rootDir, {recursive: true});
+    console.log(`Created directory ${suiteName}`);
+    createTestsuiteProperties(rootDir, suiteName);
+    createTestsuiteSuite(rootDir);
+    createTestcaseInTestsuite(rootDir);
+};
+
+export const createPackageJson = (path: string, suiteName: string) => {
+    const packageJson = {
+        name: suiteName,
+        version: "1.0.0",
+        description: "",
+        main: "index.js",
+        scripts: {
+            test: `sakuli run ${suiteName}`
+        },
+        author: "",
+        license: "ISC",
+        dependencies: {
+            "@sakuli/cli": "^2.1.3",
+        },
+    };
+
+    writeFileSync(`${path}/package.json`, JSON.stringify(packageJson, null, 4));
+    console.log("Created file package.json")
```

Comment:
Output should not be printed in this function directly. We should rather print the information in the command handler.